sdk/event_grid/Cargo.toml

```diff
@@ -13,7 +13,7 @@ categories = ["api-bindings"]
 edition = "2018"
 
 [dependencies]
-azure_core = { path = "../core", version = "0.1.0" }
+azure_core = { path = "../core", version = "0.1.0", features = ["enable_hyper"] }
```

Comment:
I'm hoping this is temporary. We can create an issue to make `event_grid` not require hyper.